src/esmock.js

```diff
@@ -15,6 +17,9 @@ const esmock = async (modulePath, mockDefs, globalDefs, opt = {}, err) => {
     .replace(/^.*:/, '') // rm windows-style drive location
     .replace(/.*at [^(]*\(/, '') // rm ' at TestContext.<anonymous> ('
 
+  if (opt.parent)
+    modulePath = fileURLToPath(new URL("../index.js", opt.parent))
```

Comment:
Shouldn't parent replace "callePath"? This would change the module being mocked, but that works correctly once it's an absolute path.

Reply:
yes you're right... updating that

Reply:
I used your recommended change but I'm not sure how to reflect that here in this conversation area

Reply:
Truthfully I'm replying from my phone; I'll review this in the morning when I can see more than a few columns at a time 😆

Reply:
haha ok thanks for messaging with me tonight